.appveyor.yml

```diff
@@ -0,0 +1,14 @@
+version: '{build}'
+os: Windows Server 2012
+matrix:
+  fast_finish: true
+
+init:
+  - cmd: choco install sbt
+  - cmd: refreshenv
+build_script:
+  - cmd: sbt dotty-bootstrapped/compile
+test_script:
+  - cmd: sbt "dotty-bootstrapped/dotc tests\pos\HelloWorld.scala"
+  - cmd: sbt sjsJUnitTests/test
+  - cmd: sbt test
```

Comment:
`test` won't work on Windows at the moment. Although `run` tests basically work, all neg tests are completely broken on Windows, because they pervasively expect file names with `/`. Fixing those is probably best left to another day, or at least another PR. ;)